src/modules/inspect/inspect.ts

```diff
@@ -117,4 +112,24 @@ export class Inspect {
 
     this._eventAggregator.publish(environment.events.inspectCorrelation.showTokenViewer, this.showTokenViewer);
   }
+
+  private async _updateInspect(diagramName: string, solutionUri: string): Promise<void> {
```

Comment:
```suggestion
  private async _updateView(diagramName: string, solutionUri: string): Promise<void> {
```

Wouldn't this be a better name here?